javaparser-core/src/main/java/com/github/javaparser/javadoc/description/JavadocInlineTag.java

```diff
@@ -97,8 +97,7 @@ public boolean equals(Object o) {
 
         JavadocInlineTag that = (JavadocInlineTag) o;
 
-        if (type != that.type) return false;
-        return content.equals(that.content);
+        return type == that.type && content.equals(that.content);
```

Comment:
The actual name is stored in the keyword attribute of the enum. I guess its alright for the equals and hashcode method. In toString we can change that. Also in JavadocBlockTag toString?